packages/react-scheduler/src/__tests__/ReactScheduler-test.js

```diff
@@ -41,16 +41,210 @@ describe('ReactScheduler', () => {
     ReactScheduler = require('react-scheduler');
   });
 
-  it('rIC calls the callback within the frame when not blocked', () => {
-    const {rIC} = ReactScheduler;
```

Comment:
It looks like lots was rewritten but really I just wrapped the existing tests in a new 'describe' block. Would separate this into a different PR ideally.

The only real change is that new tests were added at the end.